HACKING.md

```diff
@@ -433,6 +433,19 @@ maybe you need to replace system installed snap-seccomp with the one aligned to
 you are testing. To do this, simply backup `/usr/lib/snapd/snap-seccomp` and overwrite it with 
 the testing one. Don't forget to roll back to the original, after you finish testing.
 
+### Testing the snap userd agent
+
+To test the `snap userd --agent` command, you must first kill the current process if it is
+running, and then stop the dbus activation part. To do so, just run:
+
+    systemctl --user stop snapd.session-agent.socket
+
+After that, it's now possible to launch the program from a command line.
```

Comment:
(nitpick) maybe: `to launch snap userd --agent from the commandline` ?

Reply:
Good point.